docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -61,6 +61,20 @@ include::../java/GettingStarted.java[tags=showTsaApi]
 <9> Stop one Terracotta server
 <10> Get the current state of the Terracotta server
 
+== Dynamic Tsa API
+
+[source,java]
+----
+include::../java/GettingStarted.java[tags=showDynamicTsaApi]
+----
+<1> Define an intial dynamic cluster topology
```

Comment:
since this doc will be read by people migrating from fixed to dynamic config, it could be helpful to add a sentence to explain the different actions needed for a dynamic config:

something like
"A dynamic config doesn't rely on a tc-config.xml file, but is defined dynamically, therefore the Angela API provides a DynamicConfigManager.dynamicCluster() method to define within the test the different parameters.
It is possible to dynamically attach or detach stripes, which is done with the Tsa.attach() and Tsa.detach() API.
The config tool will activate attached Terracotta servers, which is done by the Tsa.activate() method"

feel free to rephrase

Reply:
Okay, will take your comment and rephrase as needed.

Reply:
Done